backend.c

```diff
@@ -1931,12 +1931,14 @@ static void *thread_main(void *data)
 	 * another thread is checking its io_u's for overlap
 	 */
 	if (td_offload_overlap(td)) {
-		int res = pthread_mutex_lock(&overlap_check);
+		int const res = pthread_mutex_lock(&overlap_check);
+		(void)res;
```

Comment:
@axboe, what do you suggest I should do here and in similar places? abort if non-zero return?